awspub/common.py

```diff
@@ -16,3 +22,35 @@ def _split_partition(val: str) -> Tuple[str, str]:
         partition = "aws"
         resource = val
     return partition, resource
+
+
+def _get_regions(s3_bucket_region: str, regions_configured: List[str]) -> List[str]:
```

Comment:
* nit: you could rename `s3_bucket_region` to `region_to_query` which would (imo) make it clearer what this region is used for
* nit: you could rename `regions_configured` to `regions_allowlist` to make it a bit clearer what this variable is about.

Reply:
Thanks for the suggestions. I've renamed those parameters, updated the docstrings and log messages.